src/pages/mypage/components/navBar/TapNavBar.tsx

```
@@ -0,0 +1,32 @@
+        onClick={() => onTabChange('genImg')}
+      >
+        <div className={styles.tapButtonText}>생성된 이미지</div>
+      </button>
+
+      <button
+        className={styles.tapButton({
+          state: activeTab === 'saveditem' ? 'active' : 'inactive',
+        })}
+        onClick={() => onTabChange('saveditem')}
+      >
+        <div className={styles.tapButtonText}>찜한 가구</div>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)